components/player.js

```diff
@@ -19,6 +19,28 @@ export default function Player () {
 
   }, [])
 
+  /*
+   * Chromecast setup
+   */
+  useEffect(() => {
+    const initializeCastApi = function() {
+      console.log('debug initializeCastApi');
+      cast.framework.CastContext.getInstance().setOptions({
+        receiverApplicationId: chrome.cast.media.DEFAULT_MEDIA_RECEIVER_APP_ID
+      });
+      const castSession = cast.framework.CastContext.getInstance().getCurrentSession();
+      console.log('debug castSession', castSession);
```

Comment:
`castSession` at this point is undefined

Reply:
have you tried requesting a session manually? 
https://developers.google.com/cast/docs/reference/web_sender/cast.framework.CastContext#requestSession

`cast.framework.CastContext.getInstance().requestSession()`